helpers/cron-requester/README.md

```diff
@@ -0,0 +1,68 @@
+ <div align="center">
+ <!-- <img align="center" width="180" src="https://i.imgur.com/62VsVXD.png" /> -->
+  <h2>Band Data Requester</h2>
+  <blockquote>A light-weight node.js tool to query data on BandChain periadically</blockquote>
+</div>
+
+## ⭐️ Features
+
+- Support cron-style scheduling
+- Support configuration via `json` file
+- No-installation required with [`npx`](https://www.npmjs.com/package/npx)
+
+## 📦 Prerequisite
+
+You need to create a `config.json` file in your machine.
+
+See example [`config.json`](./config.js) for requesting data from GuanYu [devnet](https://guanyu-devnet.cosmoscan.io/) every 5 minutes:
+
+```json
+{
+  "chainId": "band-guanyu-devnet-2",
```

Comment:
Shouldn't we get the chain ID from the new REST endpoint from #1951 instead of hard-coding it? So we don't need to always update this example file every time we re-deploy the devnet.

Reply:
this depends on `bandchain.js` support. we will patch the requester once the next event goes live.